bin/config_syntax_check.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
```

Comment:
What do people view as the long-term prospects of this script, by which I mean is it envisioned this code will be used by instructors at some point in the future, or just to be used within the build homework compilation process?